### ISSUE TITLE
Transfer all settings from local version to server 

### ISSUE THREAD

**Issue by dominijk** [apache/incubator-superset#7017]
Make sure these boxes are checked before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Superset version
Latest

### Expected results
Configure content on a local version then copy/tranfser all settings (tables, dashboards, charts,users) to a server

### Actual results
Can manual copy/paste dashboard and chart metadata but this is slow. I don't see a way to copy tables across with the parameters you've set. I think this is me missing some obvious method of doing this but can't see it in the docs. My assumption was that all of the settings are saved in the sql lite database superset makes, and that I'd just need to transfer this over.

**Comment by Avocadososemix** [comment]
This is something I have been wondering about as well. We ended up using Superset (pip install 0.29.0rc7) for a while with the default sqlite superset.db, but now that I am setting up a proper production server for Superset, I'm having trouble importing created roles/users/new permissions of the development server version of our Superset installation. When I look at our sqlite database it only contains the first two admin accouts created with fabmanager, but none of the other users/etc that were created within the superset application show up there.

**Comment by rosanzheng** [comment]
For anyone who might be still looking for a solution, here's more or less what I did (I developed locally using the PyPi setup from the installation guide):
- migrate the automatically created superset.db into a Cloud-based storage, i.e. AWS RSD MySQL
- Exchange the SQLALCHEMY_DATABASE_URI accordingly in your superset config.py
- Deploy via EC2 or a different virtual server environment, i.e. AWS EC2. For that, there are also some guides: https://www.restack.io/docs/superset-knowledge-install-superset-aws
- Add a domain, certificates, nginx to your virtual server to make it accessible accordingly

Actually, when I tried it out and tested it, I just bought very cheap server space and basically copy-pasted the local superset.db file into my virtual server through SSH. That's definetely the fastest way :-)
